### PR TITLE
Added support for Docker build of the docs/guide

### PR DESCRIPTION
**Pull request by demogorgonz**
Bellow are attached PDF files generated using docker:

[blackarch-guide-de.pdf](https://github.com/BlackArch/blackarch-guide/files/3931198/blackarch-guide-de.pdf)
[blackarch-guide-el.pdf](https://github.com/BlackArch/blackarch-guide/files/3931199/blackarch-guide-el.pdf)
[blackarch-guide-en.pdf](https://github.com/BlackArch/blackarch-guide/files/3931200/blackarch-guide-en.pdf)
[blackarch-guide-es.pdf](https://github.com/BlackArch/blackarch-guide/files/3931201/blackarch-guide-es.pdf)
[blackarch-guide-fr.pdf](https://github.com/BlackArch/blackarch-guide/files/3931202/blackarch-guide-fr.pdf)
[blackarch-guide-it.pdf](https://github.com/BlackArch/blackarch-guide/files/3931203/blackarch-guide-it.pdf)
[blackarch-guide-pt-br.pdf](https://github.com/BlackArch/blackarch-guide/files/3931204/blackarch-guide-pt-br.pdf)
[blackarch-guide-ru.pdf](https://github.com/BlackArch/blackarch-guide/files/3931205/blackarch-guide-ru.pdf)
[blackarch-guide-tr.pdf](https://github.com/BlackArch/blackarch-guide/files/3931206/blackarch-guide-tr.pdf)
[blackarch-guide-zh.pdf](https://github.com/BlackArch/blackarch-guide/files/3931207/blackarch-guide-zh.pdf)

### CODE REVIEW

**Comment by demogorgonz** [comment]
@ikstream Do you want me to change package to `texlive-full` ? The tex-live-full package is quite big [source](https://tex.stackexchange.com/questions/302676/how-large-is-the-full-install-of-texlive), so it might be a good idea to add packages separately.  Right now image is around ~1GB, while full install is, well still building for me but according to source 5GB+.

**Comment by ikstream** [comment]
adding ```texlive-full``` and some other minor packages brings me up to 3.38GB basically doubling the ~1.7 GB of your image. 
To be honest I prefer the full version to keep it simple for new contributors, who just want to add a guide.

**Comment by demogorgonz** [comment]
Sure thing, will test out and make PR within few minutes.

**Comment by ikstream** [comment]
great, thank you

File: Docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+MAINTAINER FilipS (filips@vaskir.co)
+
+#Disable interactive tzdata instalaltion
+ARG DEBIAN_FRONTEND=noninteractive
+
+
+RUN apt update && \
+    apt install -y make texlive-full latex2html texlive-latex-extra texlive-pstricks texlive-lang-* texlive-xetex
+
```
Comment (ikstream) [inline Docker/Dockerfile]: as ```texlive-full``` contains it, you can drop
```texlive-latex-extra```
```texlive-lang-*```
```texlive-pstricks```
```texlive-xetex```

https://packages.ubuntu.com/bionic/texlive-full

Comment (demogorgonz) [reply Docker/Dockerfile]: Dropped the packages, forgot to remove them :)

**Comment by ikstream** [comment]
I am currently building the docker and testing it. I expect it to work and will merge it afterwards. Thank you

**Comment by demogorgonz** [comment]
> I am currently building the docker and testing it. I expect it to work and will merge it afterwards. Thank you

Awesome, thank you. Let me know if you face any errors.

**Pull request merged**